src/ui/screens/settings/ViewTipsAndTricks.qml

```diff
@@ -155,9 +156,10 @@ VPNViewBase {
                 ColumnLayout {
                     id: layoutGuide
 
-                    anchors.fill: parent
```

Comment:
So we remove this because if the Layout doesn't require the whole window height with this it will try to stretch and then margins will get weird, correct?

Reply:
Pretty much. Inside a scroll view/flickable you should never really need to fill the parent. It can cause problems 1.) for the reason you mentioned (things align to the middle of the layout and give you weird margins/spacing) 2.) it doesn't really make sense, since the scroll view only fills the screen, but the columnlayout is going to be longer than the screen (it's more analogous to the flickables contentHeight)